recursion/problem-set-submit.checkpoint.md

```diff
@@ -0,0 +1,43 @@
+# Problem Set Submission
+
+Instructors provide code feedback through pull requests.
+
+In order to get code feedback on [Problem Set: Recursion Pt. 1](problem-set-recursion-pt-1.md) and [Problem Set: Recursion Pt. 2](problem-set-recursion-pt-2.md), follow these directions:
+
+1. Fork [this repo, `core-problem-set-recursion`](https://github.com/AdaGold/core-problem-set-recursion) to your GitHub account
+1. Clone your repo
+1. Open `recursion.py`
+
+There are comments in this file to designate where and how to put your solutions.
+
+1. **Copy and paste** your solutions from [Problem Set: Recursion Pt. 1](problem-set-recursion-pt-1.md) and [Problem Set: Recursion Pt. 2](problem-set-recursion-pt-2.md) into this file
+1. Commit and push your work to your repo
+1. Create a pull request
+1. Submit your PR URL below
+
+## Submit Work for Feedback
+
+<!-- prettier-ignore-start -->
+### !challenge
+* type: short-answer
+* id: sJEYtp
+* title: Pull Request for Feedback
+* points: 3
+* topics: python, flask
+##### !question
+
+Make a pull request against the [repo `core-problem-set-recursion`](https://github.com/AdaGold/core-problem-set-recursion) (see directions above). Place the URL of the pull request here.
+
+##### !end-question
+##### !placeholder
+
+The URL to your pull request: https://github.com/<some-ada-repo>/<project-name>/pulls/<pull-request>
+
+##### !end-placeholder
+##### !answer
+
+/.+/
+
+##### !end-answer
+### !end-challenge
+<!-- prettier-ignore-end -->
```

Comment:
Does Learn automatically add the "That’s the end! When you’re ready, submit this assessment at the top of the page." message?